documentation/modules/ROOT/pages/connectors/cassandra.adoc

```diff
@@ -983,8 +983,11 @@ the offset will be flushed every time.
 
 |[[cassandra-property-max-queue-size-in-bytes]]<<cassandra-property-max-queue-size-in-bytes, `max.queue.size.in.bytes`>>
 |`0`
-|Long value for the maximum size in bytes of the blocking queue. The feature is disabled by default, it will be active if it's set with a positive long value.
-
+|A long integer value that specifies the maximum volume of the blocking queue in bytes.
+By default, volume limits are not specified for the blocking queue.
+To specify the number of bytes that the queue can consume, set this property to a positive long value. +
+If xref:db2-property-max-queue-size[`max.queue.size`] is also set, writing to the queue is blocked when the size of the queue reaches the limit specified by either property.
```

Comment:
Should not be `db2`.

Reply:
Ugh. I'm having some copy/paste issues. :\ Sorry. I'll take care of these.